src/day-1-toy/comp.py

```diff
@@ -1,31 +1,33 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [i for i in range(1, 6)]
 
-print (y)
+# print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
+y = [i**3 for i in range(10)]
 
-print(y)
+# print(y)
 
 # Write a list comprehension to produce the uppercase version of all the
 # elements in array a. Hint: "foo".upper() is "FOO".
 
 a = ["foo", "bar", "baz"]
+y = [i.upper() for i in a]
 
-y = []
-
-print(y)
+# print(y)
 
 # Use a list comprehension to create a list containing only the _even_ elements
 # the user entered into list x.
 
-x = input("Enter comma-separated numbers: ").split(',')
-
+# print(y)
+# x = input("Enter comma-separated numbers: ").split(',')
+x = [i for i in range(20) if i%2==0]
+# print(x)
```

Comment:
Try making this work with `input()`. A hint: `input()` returns a _string_ after receiving input from the user. What type does your list comprehension need to do a modulo (`%`) operation? What can you do to change the type?